reporters/influxdb/src/main/java/io/vertx/ext/influxdb/package-info.java

```diff
@@ -0,0 +1,424 @@
+ *
+ * == Vert.x core tools metrics
+ *
+ * This section lists all the metrics generated by monitoring the Vert.x core tools.
+ *
+ * === Net Client
+ *
+ * [cols="15,50,35", options="header"]
+ * |===
+ * |Metric type
+ * |Metric name
+ * |Description
+ *
+ * |Gauge
+ * |{@code vertx.net.client.<host>:<port>.connections}
+ * |Number of connections to the remote host currently opened.
+ *
+ * |Counter
+ * |{@code vertx.net.client.<host>:<port>.bytesReceived}
+ * |Total number of bytes received from the remote host.
+ *
+ * |Counter
+ * |{@code vertx.net.client.<host>:<port>.bytesSent}
+ * |Total number of bytes sent to the remote host.
+ *
+ * |Counter
+ * |{@code vertx.net.client.<host>:<port>.errorCount}
+ * |Total number of errors.
+ *
+ * |===
+ *
+ * === HTTP Client
+ *
+ * [cols="15,50,35", options="header"]
+ * |===
+ * |Metric type
+ * |Metric name
+ * |Description
+ *
+ * |Gauge
+ * |{@code vertx.http.client.<host>:<port>.connections}
+ * |Number of connections to the remote host currently opened.
+ *
+ * |Counter
+ * |{@code vertx.http.client.<host>:<port>.bytesReceived}
+ * |Total number of bytes received from the remote host.
+ *
+ * |Counter
+ * |{@code vertx.http.client.<host>:<port>.bytesSent}
+ * |Total number of bytes sent to the remote host.
+ *
+ * |Counter
+ * |{@code vertx.http.client.<host>:<port>.errorCount}
+ * |Total number of errors.
+ *
+ * |Gauge
+ * |{@code vertx.http.client.<host>:<port>.requests}
+ * |Number of requests waiting for a response.
+ *
+ * |Counter
+ * |{@code vertx.http.client.<host>:<port>.requestCount}
+ * |Total number of requests sent.
+ *
+ * |Counter
+ * |{@code vertx.http.client.<host>:<port>.responseTime}
+ * |Cumulated response time.
+ *
+ * |Gauge
+ * |{@code vertx.http.client.<host>:<port>.wsConnections}
+ * |Number of websockets currently opened.
+ *
+ * |===
+ *
+ * === Datagram socket
+ *
+ * [cols="15,50,35", options="header"]
+ * |===
+ * |Metric type
+ * |Metric name
+ * |Description
+ *
+ * |Counter
+ * |{@code vertx.datagram.<host>:<port>.bytesReceived}
+ * |Total number of bytes received on the {@code <host>:<port>} listening address.
+ *
+ * |Counter
+ * |{@code vertx.datagram.<host>:<port>.bytesSent}
+ * |Total number of bytes sent to the remote host.
+ *
+ * |Counter
+ * |{@code vertx.datagram.errorCount}
+ * |Total number of errors.
+ *
+ * |===
+ *
+ * === Net Server
+ *
+ * [cols="15,50,35", options="header"]
+ * |===
+ * |Metric type
+ * |Metric name
+ * |Description
+ *
+ * |Gauge
+ * |{@code vertx.net.server.<host>:<port>.connections}
+ * |Number of opened connections to the Net Server listening on the {@code <host>:<port>} address.
+ *
+ * |Counter
+ * |{@code vertx.net.server.<host>:<port>.bytesReceived}
+ * |Total number of bytes received by the Net Server listening on the {@code <host>:<port>} address.
+ *
+ * |Counter
+ * |{@code vertx.net.server.<host>:<port>.bytesSent}
+ * |Total number of bytes sent to the Net Server listening on the {@code <host>:<port>} address.
```

Comment:
I guess it's "number of bytes sent **by** the Net Server"
(same applies in Hawkular doc)